dev/group_vars/apim-analytics.yml

```diff
@@ -20,6 +20,8 @@ product_name: wso2am-analytics
 product_version: 2.6.0
 target: /mnt
 product_package_location: files
+misc_file_location: "{{ product_package_location }}/misc"
+misc_file_copy_location: "{{ target }}/misc"
```

Comment:
What if the user wants to copy the files to somewhere outside the `misc` directory? Shall we remove this and have the user set the `dest` in `misc_file_list` to anywhere they want?

Reply:
If the user wants to change the place where the files are copied, they can change the value assigned to `misc_file_copy_location`.

Reply:
Refer https://github.com/wso2/ansible-apim/pull/70#discussion_r309539646